wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,41 +1,58 @@
+| ![](/wiki/shared/judgement/osu!/hit300.png "300") | 300 | The best achievable result for a hit object.  Scoring nothing except 300s in a beatmap will award the player with the SS or SSH grade. Usually denoted with a blue colour. |
+| ![](/wiki/shared/judgement/osu!/hit300g.png "Geki") | 300 (激) Geki | A variant of the 300 result. Originates from Ouendan, called Elite Beat! in EBA. Granted upon hitting a 300 on the last element in a combo in which the player has scored only 300s. Getting a Geki will grant a sizable health bonus on top of the standard health increase for a 300. |
+| ![](/wiki/shared/judgement/osu!/hit300k.png "300 Katu") | 300 (喝) Katu or Katsu | A variant of the 300 result. Originates from Ouendan, called Beat! in EBA. Granted upon hitting a 300 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 300. |
+| ![](/wiki/shared/judgement/osu!/hit100.png "100") | 100 | The second best achievable result for a hit object. Usually denoted with a green colour. |
+| ![](/wiki/shared/judgement/osu!/hit100k.png "100 Katu") | 100 (喝) Katu or Katsu | A variant of the 100 result. Originates from Ouendan, called Beat! in EBA. Granted upon hitting a 100 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 100. |
+| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
+| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
 
 ## Judgement mechanics
 
-**Hit circles:**
+### Hit circles
+
+- A hit circle is judged as a 300, 100, or 50 depending on how accurate the player's timing is when clicking on it.
+  The time intervals used to determine the concrete result for a hit circle are called *hit windows*, and are influenced by the beatmap's [overall difficulty](/wiki/Beatmap/Overall_difficulty#osu!).
+- A hit circle is judged as missed when:
+  - The circle is hit before the start of the 50 hit window and does not trigger [notelock](/wiki/Gameplay/Judgement/Notelock).
+  - The circle is hit after the end of the 50 hit window, or not hit at all.
```

Comment:
you kinda automatically get a miss if you don't hit it, and you can't really hit it afterwards

```suggestion
  - The circle is not hit and its 50 hit window passes.
```